config.yaml

```diff
@@ -384,33 +384,33 @@ Facebook:
 #  notes: ''
 #  strike_out: No
 
-#Google:
-#  career_page:
-#  office_location:
-#  office_picture:
-#  glassdoor:
-#    link:
-#    # Overwrites
-#    rating:
-#    company_size:
-#    company_revenue:
-#  software_engineer:
-#    link:
-#    # Overwrites
-#    salary:
-#    bonus:
-#    stock_options:
-#  pantry:
-#  annual_leaves:
-#  flexible_hours:
-#  benefits:
-#    good_insurance:
-#    pregnancy:
-#    maternity_leaves:
-#    covers_dependents:
-#    extras:
-#  notes: ''
-#  strike_out: No
+Google:
+ career_page: https://careers.google.com/
+ office_location: Mapletree Business City
+ office_picture: pictures/google.png
+ glassdoor: 
+   link: https://www.glassdoor.com/Reviews/Google-Reviews-E9079.htm
+   # Overwrites
+   rating: 4.4
+   company_size: 10k+
+   company_revenue: $10B+
+ software_engineer: 
+   link: https://www.glassdoor.com/Salary/Google-Software-Engineer-Singapore-Salaries-EJI_IE9079.0,6_KO7,24_IL.25,34_IN217.htm
+   # Overwrites
+   salary: $8000-9000
+   bonus: Yes$$
+   stock_options: Yes$$
+ pantry: Excellent
```

Comment:
Perhaps use Amazing since Amazing is already used for Facebook, unless this is supposed to be better than Facebook, so we will have to make it clearer. Like "More Amazing". Otherwise, if it's the same tier, we should stick with the same names so it's easily understood